docusaurus.config.ts

```diff
@@ -132,7 +132,7 @@ const config: Config = {
     },
     announcementBar: {
       id: 'announcement-bar-2', // increment on change
```

Comment:
Sidenote: If we want to show the announcement bar to everyone who already dismissed it, we need to increment this id. That said, Given that it's a minor release, I don't think we necessarily need to do it here.

Reply:
@lovromazgon TIL!